### PR TITLE
Enable Dacapo for All Platforms

### PR DESCRIPTION
**Pull request by piyush286**
- Expanded coverage from xLinux to all platforms
- Added support for zOS

Issue: https://github.com/AdoptOpenJDK/openjdk-tests/issues/1457

Signed-off-by: Piyush Gupta <piyush286@gmail.com>

### CODE REVIEW

**Comment by piyush286** [comment]
Requesting review from @smlambert.

### Testing
All builds without any additional comments below were successful, outputting `ALL TESTS PASSED`.

Platform | Jenkins
-- | --
openjdk_x86-64_linux | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1115/console
openjdk_x86-64_linux_xl | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1105/console
openjdk_aarch32_linux | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1106/console => Failed (Unrelated to this PR)       `17:04:34    gzip:   openjdk8?openjdk_impl=openj9&os=linux&arch=aarch32&release=latest&type=jdk&heap_size=normal:   not in gzip format`
openjdk_aarch64_linux | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1107/console   `Still waiting to   schedule task`
openjdk_ppc64_aix | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1108/console
openjdk_ppc64le_linux | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1109/console
openjdk_s390x_linux | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1110/console   `Still waiting to   schedule task`
openjdk_s390x_zos | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1111/console => NA       `There   are no nodes with the label   ‘ci.role.test&&hw.arch.s390x&&sw.os.zos’`
openjdk_x86-64_mac | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1112/console
openjdk_x86-64_mac_xl | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1113/console
openjdk_x86-64_windows | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1103/console
openjdk_x86-32_windows | https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1114/console

File: perf/dacapo/build.xml [hunk 1]
```diff
@@ -16,18 +16,26 @@
...
 	<property name="DEST" value="${BUILD_ROOT}/perf/dacapo" />
-	<property name="src" location="." />
+	<property name="SRC" location="." />
 	
+	<condition property="isZOS" value="true">
+		<equals arg1="${os.name}" arg2="z/OS"/>
+	</condition>
+		
+	<condition property="git-prefix" value="git" else="https">
+		<isset property="isZOS"/>
```
Comment (llxia) [inline perf/dacapo/build.xml]: I am not sure `git-prefix` is needed on zos. The below downloading link is not a git repo link.

Comment (piyush286) [reply perf/dacapo/build.xml]: Sorry, my bad! Thanks for noticing that.

**Comment by piyush286** [comment]
@llxia Updated the code!

xLinux Sanity: 
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1116/console

**Pull request merged**